vochain/scrutinizer/types/types.go

```diff
@@ -0,0 +1,171 @@
+package types
```

Comment:
Don't call this package types please, call it "indexertypes" if you want (since we'll move from scrutinizer to indexer)